### ISSUE TITLE
[1.3-rc.1] Runpath Search Paths: @executable_path/Frameworks missing for release

### ISSUE THREAD

**Issue by markoshust** [meteor/meteor#6492]
Once released through TestFlight, cordova apps fail to load. The `@executable_path/Frameworks` value needs to be added to Runpath Search Paths for `release`.

![screen-shot-2016-03-15-at-11 01 23-am](https://cloud.githubusercontent.com/assets/437029/13786322/6a1c6102-eaad-11e5-95d5-45307f1b203f.png)

**Comment by markoshust** [comment]
Ref: https://github.com/meteor/meteor/issues/6050#issuecomment-182170044

**Comment by crapthings** [comment]
+ 1

**Comment by martijnwalraven** [comment]
This turns out to be a Cordova iOS 4.0 bug, for which I've opened [an issue](https://issues.apache.org/jira/browse/CB-10885). In the meantime, I've added a workaround to the Meteor build process that should be in the next RC.

**Comment by benjamn** [comment]
Fixed in [`1.3-rc.3`](https://github.com/meteor/meteor/issues/6266#issuecomment-198569509).

**Comment by whileoneloop** [comment]
Can confirm _meteor run_ produces the correct build config.  However, _meteor build_ project output for ios appears to still miss the _Runpath Search Paths_ value "@executable_path/Frameworks" for _Release_ (in 1.3 release).  Temporary workaround: manually enter **@executable_path/Frameworks** before archiving.

**Comment by lucfranken** [comment]
Confirmed here also. Visually on the iOS device you see: The splash screen appears and then it crashes instantly. No visual errors.

This issue did not occur while running meteor run ios-device locally with a usb connected iOS device. That make it harder to reproduce. So you think you deploy a stable version but the TestFlight release will crash. 

Deploying the fix https://github.com/meteor/meteor/issues/6492#issue-141035972 works to solve the issue instantly.

**Comment by martijnwalraven** [comment]
It turned out `meteor build` would override the setting added to `build-extras.xcconfig`, which is used as a workaround until the bug is fixed in Cordova. So I changed `meteor build` to only prepare the Xcode project instead of building it (which should be done from Xcode anyway, to correctly code sign etc.). Fixed in https://github.com/meteor/meteor/commit/88d43a0f16a484a5716050cb7de8066b126c7b28, which should be in the next patch release.
